Shared/Realm.Shared/native/NativeCommon.cs

```diff
@@ -63,8 +67,18 @@ private static unsafe void DebugLogger(IntPtr utf8String, IntPtr stringLen)
 
         public static void Initialize()
         {
+            if (Environment.OSVersion.Platform == PlatformID.Win32NT)
+            {
+                var assemblyLocation = Path.GetDirectoryName(typeof(NativeCommon).Assembly.Location);
+                var architecture = Environment.Is64BitProcess ? "x64" : "x86";
+                var path = Path.Combine(assemblyLocation, "lib", "win32", architecture) + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH");
+                Environment.SetEnvironmentVariable("PATH", path);
```

Comment:
We update `PATH` so that we can point the CLR to the correct folder where it should look for `realm-wrappers.dll` based on architecture.

I opted not to use a preprocessor guard because that would be deprecated as soon as we switch to .NET Standard.